app/src/features/projects/components/ProjectSpeciesForm.tsx

```diff
@@ -4,7 +4,7 @@ import MultiAutocompleteFieldVariableSize, {
 } from 'components/fields/MultiAutocompleteFieldVariableSize';
 import { useFormikContext } from 'formik';
 import React from 'react';
-import yup from 'utils/YupSchema';
+import * as yup from 'yup';
```

Comment:
Can we revert back to the previous version (where yup is imported from the `utils/YupSchema`.  It shouldn't require any changes to anything below, it just has some of the extended yup tests I added.